app/controllers/waste_exemptions_engine/incorrect_companies_house_details_forms_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  class IncorrectCompaniesHouseDetailsFormsController < FormsController
```

Comment:
@PaulDoyle-DEFRA Is the naming of incorrect_companies_house_details okay? Wasn't sure if it was correct or not, can change the name if need to :)

Reply:
It's a bit of a mouthful. Maybe use `incorrect_company_forms_controller` like in WCR?

Reply:
Yeah that sounds better, will make the changes now, thanks! :)